### PR TITLE
Low Resolution LUT

### PR DESCRIPTION
**Pull request by ErickHernandezGutierrez**

### CODE REVIEW

File: commit/core.pyx [hunk 1]
```diff
@@ -1,837 +1,843 @@
...
+        aux = amico.lut.load_precomputed_rotation_matrices( lmax, ndirs )
+        idx_IN, idx_OUT = amico.lut.aux_structures_generate( self.scheme, lmax )
+
+        # Dispatch to the right handler for each model
+        tic = time.time()
+        self.model.generate( self.get_config('ATOMS_path'), aux, idx_IN, idx_OUT, ndirs )
+        print( '   [ %.1f seconds ]' % ( time.time() - tic ) )
+
+
+    def load_kernels( self ) :
```
Comment (MarioOcampo) [inline commit/core.pyx]: If there are kernels created with the previous version, the function `commit.core.Evaluation.load_kernels()` crashes. 

> ValueError: shapes (2640,2275) and (181,2275) not aligned: 2275 (dim 1) != 181 (dim 0)

This is an error generated in AMICO

> `KERNELS['wmr'][i,:,:] = amico.lut.resample_kernel`

**Comment by daducci** [comment]
I agree with Mario, this exception must be caught, the execution stopped and a meaningful error should be returned to the user clearly saying he/she needs to recreate the LUT

**Comment by daducci** [comment]
To be more general, I'd suggest creating a "metadata" file, e.g. `dictionary_info.pickle`, where we store all the parameters used in the `trk2dictionary` call. And `ndirs` would be just one field inside that structure. This would help us a lot in the postprocessing phase for retrieving which were the parameter values used to create that specific dictionary.

File: commit/trk2dictionary/trk2dictionary.pyx [hunk 2]
```diff
@@ -1,329 +1,361 @@
...
+    dictionary_info['peaks_use_affine'] = peaks_use_affine
+    dictionary_info['flip_peaks'] = flip_peaks
+    dictionary_info['min_seg_len'] = min_seg_len
+    dictionary_info['gen_trk'] = gen_trk
+    dictionary_info['blur_radii'] = blur_radii
+    dictionary_info['blur_samples'] = blur_samples
+    dictionary_info['blur_sigma'] = blur_sigma
+    dictionary_info['ndirs'] = ndirs
+
+    with open( filename, 'wb+' ) as dictionary_info_file:
```
Comment (MarioOcampo) [inline commit/trk2dictionary/trk2dictionary.pyx]: Move this part after the creation of  `path_out` (line 270)

File: setup.py [hunk 3]
```diff
@@ -1,48 +1,54 @@
...
+    sources=['commit/proximals.pyx'],
+    include_dirs=[numpy.get_include()],
+    extra_compile_args=['-w'],
+    extra_link_args=[],
+    language='c++',
+)
+
+setup(
+    name='commit',
+    version='1.3.0',
```
Comment (MarioOcampo) [inline setup.py]: There is a conflict with the versions. This (1.3.0) is the correct one for this implementation.

**Pull request merged**